Makefile

```diff
@@ -78,24 +63,44 @@ test_tsan: tsan          ## Run unit tests with ThreadSanitizer option.
 	./build/tests/functests -d yes ${UT_FILTER} --allow-running-no-tests ${UT_SHARD}
 	./build/mockimpl/tests_mockimpl -d yes ${UT_FILTER} --allow-running-no-tests ${UT_SHARD}
 
-.PHONY: cov     # Build unit tests with code coverage enabled.
-cov:
+.PHONY: clean
+clean:                   ## Clear build/ directory.
+	rm -rf ${DEBUG_BUILD_DIR}/*
+
+##
+## ================ integration ================
+.PHONY: fmt
+fmt:                     ## Format codes.
```

Comment:
minor: code format can move to other script file ? like: 
```Makefile
format:
	build_tools/format-diff.sh
```

Reply:
done